dags/looker.py

```diff
@@ -217,5 +217,23 @@
         **airflow_gke_prod_kwargs,
     )
 
+    delete_outdated_branches = GKEPodOperator(
+        task_id="delete_outdated_branches",
+        arguments=[
+            "python",
+            "-m",
+            "looker_utils.main",
+            "delete-branches",
+            "--n_days 180"
+        ],
+        image="gcr.io/moz-fx-data-airflow-prod-88e0/looker-utils_docker_etl:latest",
+        dag=dag,
```

Comment:
Passing in the DAG as an argument isn't necessary since we're within a `with DAG(...):` context block.
```suggestion
```